### ISSUE TITLE
Incorrect line for cloning Seshat repository

### ISSUE THREAD

**Issue by btasde42** [bootphon/seshat#4]
git clone --recurse-submodules git@github.com:bootphon/seshat.git

This line raises an "fatal: Could not read from remote repository." error.

It should be changed to "git clone --recurse-submodules https://github.com/bootphon/seshat.git "

**Comment by hadware** [comment]
Yeah i'll fix that. This is an artefact of when we used to host the code on our private gitlab repository.
